backend/app/alembic/versions/d4867f3a4c0a_first_revision.py

```diff
@@ -23,38 +24,22 @@ def upgrade():
         sa.Column("id", sa.Integer(), nullable=False),
         sa.Column("full_name", sa.String(), nullable=True),
         sa.Column("email", sa.String(), nullable=True),
-        sa.Column("hashed_password", sa.String(), nullable=True),
+        sa.Column("profile_picture", sa.String(), nullable=True),
+        sa.Column("hashed_password", sa.String(), nullable=True, default=None),
```

Comment:
Why add the `default=None` for this?

Reply:
oops, wanted to add that to profile_picture

Reply:
Figured